docs/articles/blog.md

```diff
@@ -0,0 +1,131 @@
+
+A<sup>2</sup>I<sup>2</sup> is Australia's largest AI institute recently formed at Deakin University. Our team specialises in building early stage prototypes that are transferred to industry for evaluation and ongoing support. Due to the variety and number of projects we work on (more details [here](http://a2i2.ai)) we needed a flexible and consistent way to organise our machine learning projects. Surround was developed to help improve **reliability**, **maintainability**, **reproducibility** and **consistency** in our projects.
+
+Surround is now available to the community for feedback and is actively being developed [http://github.com/dstil/surround](http://github.com/dstil/surround).
+
+## Motivation
+
+**Huge effort is required to deploy an early prototype into a production environment**
+
+Typically, a project is first given to a research engineer to develop the initial solution (prepare the data, select an appropriate algorithm, train a model, evaluate the approach etc.) A software engineer is then tasked with preparing the scripts for integration and deployment into a client's environment. We found that many concerns of a production system such as how will configuration for different environments (development, staging and production) be handled, what is the format and location of the log files, will a queue or web interface be used etc. all impact how the initial solution is implemented. None of these concerns are paramount in the research engineers mind who typical come from a scientific background rather than an engineering background. Resolving these challenges ensuring the needs of the whole team (research engineers and software engineers) are satisfied is time consuming and must be repeated for each client project.
+
+**Maintaining a production-grade machine learning system is hard**
+
+Over time we realised that clients were finding it difficult to maintain and support such complex pieces of technology. And no wonder as do do the tech giants like [Google](https://ai.google/research/pubs/pub46555). We also noticed that while each solution is unique there are many aspects which are the same but that there are no opensource convention based frameworks for developing machine learning projects. Unlike say web development which abounds in frameworks such as Ruby on Rails, Spring Boot, Laravel, Django etc. We hope to build a community around Surround that is focused on building best practices into a single framework to improve the quality of the systems we build and increase developer productivity.
```

Comment:
wonder as do do the tech giants > this is a sentiment echoed by the tech giants?